docs/contributor_guide/seed_scenarios.rst

```diff
@@ -0,0 +1,85 @@
+`Hutchinson et al. (2020) <https://arxiv.org/pdf/2005.00813.pdf>`_
+
+Employment evaluations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A potential client asks you if ML can help with predicting job candidates' suitability for jobs based on a combination of personality tests and body language
+`Rhagavan et al. (2019) <https://arxiv.org/pdf/1906.09208.pdf>`_
+
+Rankings for image search
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're on a team working on improving an image search system after receiving some complaints from users related to fairness.  Users often use this system to find a selection of stock images to use when making multimedia presentations.  In this system, requests start with some information about the context and user creating the query, and your team is trying to incorporate ideas about fairness like diversity and inclusision into how search results are ranked.
+`Mitchell et al. (2020) <https://arxiv.org/pdf/2002.03256.pdf>`_
+
+Sales leads for car loans
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You work at CarCorp, a company that collects special financing data: information on people who need car financing but have either low credit scores or limited credit histories, and sells this data to auto dealer as sales leads.  CarCorp  serves dealers across the United States.  A new project manager asks about leveraging data science to “improve the quality” of leads so that dealers to not churn.  CarCorp has a large amount of historical lead data (2 million leads in 2017 alone), but relatively less data on which leads had been approved for special financing (let alone why the loan was approved).
+`Passi and Barocas (2019) shttps://arxiv.org/ftp/arxiv/papers/1901/1901.02547.pdf>`_
+
+Predictive policing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You are a contractor working with the police department in a large city.  One of the project leaders in the department would like to construct a risk score for people who are known gang members engaging in knife crime.  It's important to them that they can understand what the model is doing, and are way that any model will pick up on protected characteristics.
+`Veale et al. (2018) <https://arxiv.org/pdf/1802.01029.pdf>`_
+
+Scheduling maintenance within a factory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You work within a manufacturing company, and are starting a new project that will create a schedule assigning employees to check and update certain components of the machinery to prevent critical operation failures. The component assignment is based on data that show how often different components have worn out and broken down in the past.
+`Kyung Lee (2018) <AlgoManagePerception.pdf>`_
```

Comment:
This link won't work, but I'm sure we can find the original one easily.

Is the point here that the created schedule should be fair?

Reply:
@romanlutz thanks!  I fixed up the link.

This is my intention:
> Perhaps we could use these ~15 seed scenarios (and any ideas other folks have!) as a starting point for scanning through, voting, and picking a few deployment contexts to explore further.

> Once there's a concrete deployment context, we can sketch out "what kinds of system behaviors are harmful, in what ways, to whom, and why, including the normative reasoning underlying these statements" (Blodget et al. 2020), and then explore how fairlearn could contribute and work that into a notebook that meets the aspirations in #490.

I hear you thinking ahead about what fairness means in this context, which seems great!  👍 If we decide to move forward with this one, and it's enough of a **Real deployment context**, then we could move on to discussing **Real harms** as in #490.

Reply:
Sure, I just don't think we can vote without  understanding what it's actually about (roughly at least).

Reply:
@romanlutz It's a great question to ask!  But I don't think it'll be possible for everyone to have exactly the same understanding.  I think we have to trust that everyone in the conversation can make reasonable votes, given their diverse perspectives and experiences. :)

Concretely, if this seed scenario doesn't make sense to you in terms of  #490, you can decide whether it's better to dig in and read the paper and ask questions more on Thursday to see what other folks think.  Or you could decide to just ignore this one and focus that energy on other seed scenarios that seem more promising to you in terms of #490.  As long as everyone's mindful of bikeshedding, I hope everyone brings their own perspectives on what is important! 👍